flask/src/main.py

```diff
@@ -96,6 +99,39 @@ def __init__(self, import_name, *args, **kwargs):
 app = MyFlask(__name__)
 CORS(app)
 
+client = OpenAI(api_key= os.environ["OPENAI_API_KEY"])
```

Comment:
you need to use `os.getenv()`
`os.environ[]` raises KeyError when variable not set
as is it would break production (and local when API key not set)

Reply:
and the other spot too
and i think this needs to be moved into a route so we don't initialize OpenAI with None for every request that's not /suggestion

Reply:
the client just get's initialized once right? not on every route request

Reply:
probably... not sure how gunicorn manages its objects
but regardless of whether it's once or not it will fail to initialize if key is None (which we decided will be the case in non-local env)

Reply:
if you want it to be more efficient you can put it under a conditional that checks if key exists instead of putting it inside the route

Reply:
Putting it in the route, simpler.